seaborn/tests/test_axisgrid.py

```diff
@@ -436,6 +436,7 @@ def test_set_titles_margin_titles(self):
         nt.assert_equal(g.axes[0, 1].get_title(), "b == n")
         nt.assert_equal(g.axes[1, 0].get_title(), "")
 
+    @pytest.mark.xfail(mpl.__version__ == "3.1.0", reason="matplotlib bug")
```

Comment:
This also fails with matplotlib 3.1.1, so you'll probably want to disable it there, too.

Do you know if the bug here is in matplotlib or seaborn?